libs/s25main/ingameWindows/iwBuildOrder.cpp

```diff
@@ -4,25 +4,44 @@
 
 #include "iwBuildOrder.h"
 #include "GamePlayer.h"
+#include "GlobalGameSettings.h"
 #include "Loader.h"
 #include "WindowManager.h"
+#include "WineLoader.h"
+#include "addons/const_addons.h"
 #include "controls/ctrlComboBox.h"
 #include "controls/ctrlImage.h"
 #include "controls/ctrlList.h"
 #include "network/GameClient.h"
+#include "world/GameWorldBase.h"
 #include "world/GameWorldViewer.h"
 #include "gameData/BuildingConsts.h"
 #include "gameData/const_gui_ids.h"
 
+void iwBuildOrder::fillBuildOrderAndRemoveUnusedBuildings()
```

Comment:
The caller doesn't need to know about the removing, that's an internal detail of this function. So keep it short:
```suggestion
void iwBuildOrder::fillBuildOrder()
```